.github/workflows/jsr.yml

```diff
@@ -0,0 +1,31 @@
+name: JSR
+
+on:
+  pull_request:
+    paths:
+      - "packages/sdk-js/**"
```

Comment:
This will publish before we merge to main, which I don't think we want.

Reply:
Good call! 

This should run when there's a push to main && when there are changes to `/sdk-js`. Does that seem right?
```
on:
  push:
    branches:
      - main
    paths:
      - "packages/sdk-js/**"
```